MantidPlot/src/ProjectRecoveryGUIs/ProjectRecoveryView.h

```diff
@@ -8,20 +8,17 @@
 #define PROJECTRECOVERYVIEW_H
 
 #include "ProjectRecoveryPresenter.h"
+#include "ui_ProjectRecoveryWidget.h"
```

Comment:
I think you can now remove this include from `ProjectRecoveryView.cpp`. Also, please check the other header/source files which have conversions to `unique_ptr` for similar issues.

Reply:
I know it's not nessercary but, shouldn't the .cpp be able to be independant of changes in the .h with regards to includes, for example if the header no longer needs #include "ui_ProjectRecoveryWidget.h" and it is removed then it would throw an error, whilst in this case it wouldn't. At least it seems like better practice to me.

Reply:
There is indeed some sense behind the rule you mention. However, we follow the LLVM coding standards, and they state: '#include hurts compile time performance. Don’t do it unless you have to, especially in header files.' See [here](http://llvm.org/docs/CodingStandards.html#include-as-little-as-possible).

Reply:
That's interesting, I will follow that from now on! :+1: